packages/panw_cortex_xdr/data_stream/incidents/elasticsearch/ingest_pipeline/default.yml

```
@@ -0,0 +1,212 @@
+---
+description: Pipeline for Palo Alto XDR Incident API.
+processors:
+  - set:
+      field: ecs.version
+      value: '8.9.0'
+  - set:
+      field: event.kind
+      value: alert
+  - append:
+      field: event.category
+      value: malware
+  - append:
+      field: event.type
+      value: info
+  - rename:
+      field: message
+      target_field: event.original
+  - json:
+      field: event.original
+      target_field: panw_cortex.xdr
+  - drop:
+      if: ctx.panw_cortex?.xdr?.reply?.result_count != null && ctx.panw_cortex?.xdr?.reply?.result_count == 0
+  - fingerprint:
+      fields:
+        - panw_cortex.xdr.alert_count
+        - panw_cortex.xdr.incident_id
+      target_field: "_id"
+      ignore_missing: true
+  - date:
+      field: panw_cortex.xdr.creation_time
+      target_field: panw_cortex.xdr.creation_time
+      formats:
+        - UNIX_MS
+      if: ctx.panw_cortex?.xdr?.creation_time != null
+  - date:
+      field: panw_cortex.xdr.resolved_timestamp
+      target_field: panw_cortex.xdr.resolved_timestamp
+      formats:
+        - UNIX_MS
+      if: ctx.panw_cortex?.xdr?.resolved_timestamp != null
+  - date:
+      field: panw_cortex.xdr.modification_time
+      target_field: panw_cortex.xdr.modification_time
+      formats:
+        - UNIX_MS
+      if: ctx.panw_cortex?.xdr?.modification_time != null
+  - set:
+      field: "@timestamp"
+      copy_from: panw_cortex.xdr.creation_time
+      if: ctx.panw_cortex?.xdr?.creation_time != null
+  - set:
+      field: event.severity
+      value: 0
+      if: ctx.panw_cortex?.xdr?.severity == "unknown"
+  - set:
+      field: event.severity
+      value: 1
+      if: ctx.panw_cortex?.xdr?.severity == "informational"
+  - set:
+      field: event.severity
+      value: 2
+      if: ctx.panw_cortex?.xdr?.severity == "low"
+  - set:
+      field: event.severity
+      value: 3
+      if: ctx.panw_cortex?.xdr?.severity == "medium"
+  - set:
+      field: event.severity
+      value: 4
+      if: ctx.panw_cortex?.xdr?.severity == "high"
+  - rename:
+      field: panw_cortex.xdr.incident_id
+      target_field: event.id
+      ignore_missing: true
+  - rename:
+      field: panw_cortex.xdr.description
+      target_field: event.reason
+      ignore_missing: true
+      if: "ctx.panw_cortex?.xdr?.description != null && ctx.panw_cortex.xdr.description instanceof String"
+  - set:
+      field: related.hosts
+      copy_from: panw_cortex.xdr.hosts
+      if: ctx.panw_cortex?.xdr?.hosts != null
+  - set:
+      field: related.user
+      copy_from: panw_cortex.xdr.users
+      if: ctx.panw_cortex?.xdr?.users != null
+  - gsub:
+      field: related.hosts
+      pattern: ":.*"
+      replacement: ""
+      if: ctx.related?.hosts != null
+  - gsub:
+      field: related.user
+      pattern: ".*\\\\"
```

```suggestion
      pattern: '.*\\'
```
Let's not proliferate `\`, the world already has far too many.